src/main/java/org/jabref/gui/preferences/PreferencesDialog.java

```diff
@@ -127,6 +128,12 @@ private void construct() {
                                                 .withText(PrefsTab::getTabName)
                                                 .install(tabsList);
 
+        TextField searchBox = new TextField();
+        searchBox.setPromptText(Localization.lang("Search"));
+
+        PreferencesSearchHandler searchHandler = new PreferencesSearchHandler(preferenceTabs, searchBox.textProperty());
```

Comment:
Bind the searchBox.textProperty the same ways as you do with the itemsProperty below